website/docs/docs/collaborate/data-tile.md

```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
 
 If your analytics tool supports iFrames, you can embed the dashboard tile within it. 
 
-#### Tableau example
+### Examples
+The following examples show how to embed the data health tile in Tableau and PowerBI.
+
+<Tabs>
+
+<TabItem value="powerbi" label="PowerBI example">
+
+You can embed the data health tile iFrame in PowerBI using PowerBI Pro Online, Fabric PowerBI, or PowerBI Desktop. 
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi.png" width="80%" title="Embed data health tile iFrame in PowerBI"/>
+
+Follow these steps to embed the data health tile in PowerBI:
+
+1. Create a dashboard in PowerBI and connect to your database to pull in the data.
+2. Create a new measurement with the iFrame.
```

Comment:
what does this mean specifically? is this something done in powerbit or explorer?